src/packages/fuelrats_api/v3/converters.py

```diff
@@ -1,11 +1,13 @@
 from __future__ import annotations
 
 import json
-from datetime import datetime
+
+import pendulum
+from pendulum import DateTime
 from typing import Union, Any
 from uuid import UUID
 
-from dateutil.parser import parse as datetime_parser
+from dateutil.parser import parse as DateTime_parser
```

Comment:
[error] don't think this is necessary any more
[style] this shouldn't have been renamed ;)